docs/Other SDL Features/Playing Audio Indications/index.md

```diff
@@ -23,12 +20,26 @@ sdlManager.fileManager.upload(file: audioFile) { (success, bytesAvailable, error
     <#audio file is ready if success is true#>
 }
 ```
+!@
+
+@![android, javaSE, javaEE]
+```java
+SdlFile audioFile = new SdlFile("Audio File Name", FileType.AUDIO_MP3, Uri.parse("File Location"), true);
+sdlManager.getFileManager().uploadFile(audioFile, new CompletionListener() {
+	@Override
+	public void onComplete(boolean success) {
+
+	}
+});
+```
+!@
 
-For more information about uploading files, see [the relevant guide](Other SDL Features/Uploading Files).
+For more information about uploading files, see the [Uploading Files guide](Other SDL Features/Uploading Files).
 
 ## Using the Audio File in an Alert
-Now that the file is uploaded to the remote system, it can be used in various APIs, such as `Speak`, `Alert`, `AlertManeuver`, `PerformInteraction`. To use the audio file in an alert, you simply need to construct a `TTSChunk` referring to the file's name.
+Now that the file is uploaded to the remote system, it can be used in various RPCs, such as `Speak`, `Alert`, and `AlertManeuver`. To use the audio file in an alert, you simply need to construct a @![iOS]`SDLTTSChunk`!@ @![android, javaSE, javaEE]`TTSChunk`!@ referring to the file's name.
```

Comment:
```suggestion
Now that the file is uploaded to the remote system, it can be used in various RPCs, such as `Speak`, `Alert`, and `AlertManeuver`. To use the audio file in an alert, you simply need to construct a @![iOS]`SDLTTSChunk`!@@![android, javaSE, javaEE]`TTSChunk`!@ referring to the file's name.
```